### PR TITLE
docs: add latest finschia mainnet/testnet information

### PR DESCRIPTION
**Pull request by jaeseung-bae**
## Description

* Add latest finschia mainnet, ebony testnet information including below
  * Public endpoints
  * Genesis file download links
  * Testnet Faucet information
  * Example CLIs and REST APIs

* View: https://github.com/jaeseung-bae/finschia/blob/docs/finschia-io/README.md

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (build) [\#395](https://github.com/Finschia/finschia/pull/395) Apply Go 1.22, finschia-sdk 0.49.1 and update outdated dependencies
 
 ### Docs
+* (doc) [\#400](https://github.com/Finschia/finschia/pull/400) Add  latest finschia mainnet, ebony testnet information
```
Comment (tkxkd0159) [inline CHANGELOG.md:67]:
```suggestion
* (docs) [\#400](https://github.com/Finschia/finschia/pull/400) Add  latest finschia mainnet, ebony testnet information
```

**Review by tkxkd0159 — APPROVED** [review]

**Review by 0Tech — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -83,7 +81,108 @@ make localnet-start
...
+* Genesis file
+    * Finschia mainnet
+        * [finschia-genesis.tgz](https://vos.line-scdn.net/finschia-2-fileshare/datafile/finschia-prod-2/finschia-2-genesis.tgz)
+    * Ebony testnet
+        * [ebony-genesis.tgz](https://vos.line-scdn.net/finschia-2-fileshare/ebony-prod-2/genesis-file.tgz)
+* Snapshot file
+    * Finschia mainnet (pruned)
+        * [finschia-2-pruned.tgz](https://finschia-quicksync.line-scdn.net/finschia-2/pruned/finschia-2-pruned.tgz)
+    * Ebony testnet (pruned)
+        * [ebony-2-pruned.tgz](https://finschia-quicksync.line-scdn.net/ebony-2/pruned/ebony-2-pruned.tgz)
```
Comment (zemyblue) [inline README.md:104]: How about adding the data size if possible?

Comment (jaeseung-bae) [reply README.md:104]: Of course, It's possible.
But it's little bit redundant, web browsers show us the size and progress when we click the download link.

Comment (jaeseung-bae) [reply README.md:104]: And size information is static, that could be outdated when we update the file.

**Pull request merged**